src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -5,60 +5,97 @@
+                  :width               32
+                  :style               {:margin-right 20}
+                  :accessibility-label :options-button
+                  :on-press            #(rf/dispatch [:bottom-sheet/show-sheet
+                                                      {:content (fn [] [actions/group-details-actions group])}])}
+     [quo2/icon :i/options {:color (colors/theme-colors colors/neutral-100 colors/white)}]]))
+
+(defn top-buttons []
+  [rn/view {:style {:flex-direction     :row
+                    :padding-horizontal 20
+                    :justify-content    :space-between}}
+   [back-button] [options-button]])
 
 (defn count-container [count]
   [rn/view {:style (style/count-container)}
    [quo2/text {:size   :label
                :weight :medium
                :style  {:text-align :center}} count]])
 
-(defn prepare-members [members]
-  (let [admins (filter :admin? members)
-        online (filter #(and (not (:admin? %)) (:online? %)) members)
-        offline (filter #(and (not (:admin? %)) (not (:online? %))) members)]
-    (vals (cond-> {}
-            (seq admins) (assoc :owner {:title "Owner" :data admins})
-            (seq online) (assoc :online {:title "Online" :data online})
-            (seq offline) (assoc :offline {:title "Offline" :data offline})))))
-
 (defn contacts-section-header [{:keys [title]}]
   [rn/view {:style {:padding-horizontal 20 :border-top-width 1 :border-top-color colors/neutral-20 :padding-vertical 8 :margin-top 8}}
    [quo2/text {:size   :paragraph-2
                :weight :medium
                :style  {:color (colors/theme-colors colors/neutral-50 colors/neutral-40)}} title]])
 
+(def added (reagent/atom ()))
+
+(defn contact-requests-sheet [group]
+  [:f>
+   (fn []
+     (let [{window-height :height} (rn/use-window-dimensions)
+           safe-area (safe-area/use-safe-area)]
+       [rn/view {:style {:height (- window-height (:top safe-area))}}
+        [rn/touchable-opacity
+         {:on-press #(rf/dispatch [:bottom-sheet/hide])
+          :style (style/close-icon)}
+         [quo2/icon :i/close {:color (colors/theme-colors colors/neutral-100 colors/white)}]]
+        [quo2/text {:size   :heading-1
+                    :weight :semi-bold
+                    :style  {:margin-left 20}}
+         (i18n/label :t/add-members)]
+        [rn/text-input {:placeholder (str (i18n/label :t/search) "...")
+                        :style       {:height             32
+                                      :padding-horizontal 20
+                                      :margin-vertical    12}
+                        :on-change   (fn [e] (rf/dispatch [:contacts/set-search-query (oget e "nativeEvent.text")]))}]
+        [contact-list/contact-list {:icon  :check
+                                    :group group
+                                    :added added
+                                    :search? true}]
+        [rn/view {:style (style/bottom-container safe-area)}
```

Comment:
bottom-sheet should take care of that, you don't need to use here safe-area, if it looks wrong means bottom sheet is broken, but new version is being developed